src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/opentelemetry-collector-infra-hosts.mdx

```diff
@@ -0,0 +1,241 @@
+---
+title: Collector for host monitoring
+tags:
+  - Integrations
+  - Open source telemetry integrations
+  - OpenTelemetry
+metaDescription: The OpenTelemetry collector is a central tool to collect, process, and export your telemetry.
+---
+
+You can collect metrics about your infrastructure hosts with OpenTelemetry if you set up a collector. The collector is a component of OpenTelemetry that collects, processes, and exports telemetry data to New Relic (or any observability back-end).
+
+The collector setup is part of the larger process of setting up OpenTelemetry with New Relic. Before following the collector steps below, make sure you've completed the [preceding setup steps](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-quick-start).
+
+If you're looking for help with other collector use cases, see the [newrelic-opentelemetry-examples](https://github.com/newrelic/newrelic-opentelemetry-examples) repository.
+
+## Step 1: Install the OpenTelemetry collector [#install-generic]
+
+To do a basic installation for single hosts in the cloud or on-premises, see [OpenTelemetry's instructions](https://opentelemetry.io/docs/collector/getting-started/#linux-packaging) for up-to-date installation steps from the community. Instructions are available for the following:
+
+* Linux: Debian systems
+* Linux: Red Hat
+* Windows
+* Docker, Kubernetes, and other options
+
+Your deployment experience might vary depending on which vendor-specific distributions you use. For example, installation via a package manager might be available for Linux hosts.
+
+<Callout variant="important">
+To set up infrastructure monitoring, you need to install and configuring components that are included in the `collector-contrib` release. For example, the host receiver is required to collect basic host metrics such as CPU, memory, disk and network stats and is only available in the [OpenTelemetry collector-contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib) release.
```

Comment:
One collector-contrib reference in this paragraph has backticks, the other does not. I don't care which you use, but I do like consistency! :-)

Reply:
Barb, I think the instance without back-ticks is a link, so it may look unusual to have the back-ticks inside the link.